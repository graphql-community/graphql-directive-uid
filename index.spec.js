import directive from './index';

it('getDirectiveDeclaration should be defined', () => {
  expect(directive.getDirectiveDeclaration()).toMatchSnapshot();
});
