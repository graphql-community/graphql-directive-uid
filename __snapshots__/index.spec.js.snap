// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDirectiveDeclaration should be defined 1`] = `
GraphQLDirective {
  "args": Array [
    Object {
      "astNode": undefined,
      "defaultValue": undefined,
      "description": null,
      "name": "from",
      "type": "[String]",
    },
  ],
  "astNode": undefined,
  "description": "Generates unique ID based on specifics fields",
  "locations": Array [
    "OBJECT",
  ],
  "name": "uid",
}
`;
